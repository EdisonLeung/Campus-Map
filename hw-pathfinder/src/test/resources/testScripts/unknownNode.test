# test functionality for pathfinding with a node that graph does
# not contain

# Create graph
CreateGraph unknown

# Add nodes
AddNode unknown a
AddNode unknown b

# Add edge
AddEdge unknown a b 1.0

# test pathfinding
FindPath unknown x b

FindPath unknown a y

FindPath unknown x y