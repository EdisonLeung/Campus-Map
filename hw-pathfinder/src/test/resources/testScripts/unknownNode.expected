# test functionality for pathfinding with a node that graph does
# not contain

# Create graph
created graph unknown

# Add nodes
added node a to unknown
added node b to unknown

# Add edge
added edge 1.000 from a to b in unknown

# test pathfinding
unknown: x

unknown: y

unknown: x
unknown: y